src/components/Scoreboard.tsx

```diff
@@ -155,9 +153,9 @@ const styles = StyleSheet.create({
 		bottom: 1,
 		flexDirection: "row",
 		flexWrap: "wrap",
-		justifyContent: "space-between",
+		justifyContent: "space-evenly",
 		width: wp("100%"),
-		height: hp("27.5%"),
+		height: hp("25.8%"),
 		backgroundColor: "#34558b",
```

Comment:
looks good, a nit: i'd add some comments in the code prior to methods , etc just good practice. At the very least top level comments describing the component

Reply:
Done!